koku/api/settings/tags/mapping/view.py

```diff
@@ -35,6 +34,15 @@
 from reporting.provider.all.models import TagMapping
 
 
+def parse_list(string_to_parse):
```

Comment:
This is converting a string to a list. The current name makes me think it is operating on a list. Type hints would make it clearer.

```suggestion
def parse_list(string_to_parse: str) -> list[str]:
```